send.sh

```diff
@@ -31,6 +31,16 @@ COMMITTER_NAME="$(git log -1 "$TRAVIS_COMMIT" --pretty="%cN")"
 COMMIT_SUBJECT="$(git log -1 "$TRAVIS_COMMIT" --pretty="%s")"
 COMMIT_MESSAGE="$(git log -1 "$TRAVIS_COMMIT" --pretty="%b")" | sed -E ':a;N;$!ba;s/\r{0,1}\n/\\n/g'
 
+if [ ${#COMMIT_SUBJECT} -gt 256 ]; then
+  COMMIT_SUBJECT="$(echo "$COMMIT_SUBJECT" | cut -c 1-250)"
```

Comment:
```suggestion
  COMMIT_SUBJECT="$(echo "$COMMIT_SUBJECT" | cut -c 1-253)"
```
We can have it sliced up to 253 chars and then the `...` would be 3 more, which would be exactly 256.

Reply:
I thought about it. It was just in case. No problem for me.